src/hub.js

```diff
@@ -154,6 +154,10 @@ const mediaSearchStore = window.APP.mediaSearchStore;
 const OAUTH_FLOW_PERMS_TOKEN_KEY = "ret-oauth-flow-perms-token";
 const NOISY_OCCUPANT_COUNT = 12; // Above this # of occupants, we stop posting join/leaves/renames
 
+// Maximum number of people in the room/entering before users are forced to observer mode.
+// Eventually this should be moved to a room setting.
+const MAX_OCCUPIED_ROOM_ENTRY_SLOTS = 24;
```

Comment:
Would it be useful to make this optionally configurable with a query string param, for debugging at least?

Reply:
good call

Reply:
on second thought, I think this might be a bad idea since it makes it trivial to bypass this check. not going to argue this check is the most robust thing in the world, but having it be reduced down to 'someone can just share a URL to bypass it' seems excessively weak.

Reply:
i think insofar as we get complaints about this, we should escalate pushing this into a room setting. i was going to do that for v1 but didn't want to get bogged down since I'd like to get this out ahead of tomorrow's event.